### PR TITLE
Issue 5695: Sign Windows Editor for stable releases

### LINKED ISSUE #5695: Fix signing of stable editor releases
The Editor Windows releases from the stable (master) branch weren't signed.

### PR DESCRIPTION
**Pull request by JCash**

### CODE REVIEW

File: .github/workflows/main-ci.yml [hunk 1]
```diff
@@ -305,21 +305,31 @@ jobs:
       { name: 'Install Java', uses: actions/setup-java@v1, with: { java-version: '11.0.2' } },
       {
         name: 'Download editor',
+        shell: bash,
```
Comment (JCash) [inline .github/workflows/main-ci.yml:308]: Last time I fixed this, I spent a lot of time trying to figure out how to run the scripts in PowerShell.

It turned out then, that the bash shell worked well, it was just a problem with the encoding of the cert from the b64 Github secret into the cert file.

And, after those fixes, I also forgot to add it in the main-ci.yml script.
This version is used as the base version for each release, and is the one the users are most likely to download from the web.

File: .github/workflows/main-ci.yml [hunk 2]
```diff
@@ -305,21 +305,31 @@ jobs:
...
         if: (github.event_name == 'push') || (github.event_name == 'pull_request_target') || ((github.event_name == 'repository_dispatch') && (github.event.client_payload.skip_sign != true)),
         run: 'ci/ci.sh download-editor --platform=${{ matrix.platform }}'
       },
       {
         name: 'Install dependencies',
+        shell: bash,
+        if: (github.event_name == 'push') || (github.event_name == 'pull_request_target') || ((github.event_name == 'repository_dispatch') && (github.event.client_payload.skip_sign != true)),
+        run: 'ci/ci.sh --windows-cert-b64="${{env.WINDOWS_CERTIFICATE}}" --windows-cert-pass="${{env.WINDOWS_CERTIFICATE_PASS}}" install'
```
Comment (JCash) [inline .github/workflows/main-ci.yml:316]: Use base64 encoded cert

File: .github/workflows/main-ci.yml [hunk 3]
```diff
@@ -305,21 +305,31 @@ jobs:
...
+      },
+      {
+        name: 'Install ext dependencies',
+        shell: bash,
         if: (github.event_name == 'push') || (github.event_name == 'pull_request_target') || ((github.event_name == 'repository_dispatch') && (github.event.client_payload.skip_sign != true)),
-        run: 'ci/ci.sh --windows-cert="${{env.WINDOWS_CERTIFICATE}}" --windows-cert-pass="${{env.WINDOWS_CERTIFICATE_PASS}}" install'
+        run: 'ci/ci.sh install_ext --platform=${{ matrix.platform }}'
```
Comment (JCash) [inline .github/workflows/main-ci.yml:322]: Install the signtool.exe from out own packages.

**Review by britzl — DISMISSED** [review]
Looks good!

**Pull request merged**